rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/LRAParticipantRecord.java

```diff
@@ -1000,4 +1033,16 @@ private String getLRAId(URI lraId) {
 
         return path.substring(path.lastIndexOf('/') + 1);
     }
+
+    private void trace_progress(String reason) {
+        if (LRALogger.logger.isTraceEnabled()) {
+            LRALogger.logger.tracef("%s: Participant id: %s, LRA id: %s, reason: %s, state: %s, accepted: %b",
+                    LocalDateTime.now(ZoneOffset.UTC), // use the same time function as used for LRA timeouts
+                    lraId,
+                    participantPath,
+                    reason,
+                    status,
+                    accepted);
+        }
+    }
```

Comment:
WDYT about moving `trace_progress` to `io.narayana.lra.logging`? For example, adding it to `LRALogger` or creating a new utility class?

Reply:
It's just a trace statement, there is no desire to over-engineer this.